tck-runner/pom.xml

```diff
@@ -323,5 +320,47 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>security-manager</id>
+            <activation>
+                <property>
+                    <name>with-security-manager</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <arquillian.protocol>LocalSecurityManagerTesting</arquillian.protocol>
```

Comment:
How does this work? Do you define a new protocol? Is this an Arquillian feature? Wouldn't mind somewhere a link/hint to some more information